src/hooks/useWalletConnectEventsManager.ts

```diff
@@ -137,7 +154,7 @@ export default function useWalletConnectEventsManager(initialized: boolean) {
 			// signClient.on('session_ping', data => console.log('ping', data))
 			// signClient.on('session_event', data => console.log('event', data))
 			// signClient.on('session_update', data => console.log('update', data))
-			// signClient.on('session_delete', data => console.log('delete', data))
+			web3wallet.on('session_delete', onSessionDeleted)
```

Comment:
Nice ! 

I think we should handle others functions like `session_ping ` or `session_update `